ledger/metrics/src/main/scala/com/daml/metrics/api/MetricHandle.scala

```diff
@@ -19,24 +19,32 @@ object MetricHandle {
 
   trait Factory {
 
-    def prefix: MetricName
```

Comment:
Is there any reason behind removing `prefix` besides simplifying the `Factory` API?